src/librustc_lsan/lib.rs

```diff
@@ -0,0 +1,6 @@
+#![cfg_attr(not(stage0), feature(sanitizer_runtime))]
```

Comment:
I suppose this crate should also contain the `-lpthread -lrt -lm` linker flags that runtimes supossedly depend on. But I'm not sure what runtime depends on what library.